### DISCUSSION TITLE
Acessibility Feature for old-style flinching

### DISCUSSION THREAD

**Discussion by ghost** [SmartlyDressedGames/Unturned-3.x-Community#4675] [Ideas]
i have vivid memory of this being a thing, or being suggested, but im not sure fully.
new flinch system is a problem for a demographic of people who have issues with motion sickness. i remember that this was mentioned before that this was gonna be a thing but i think we all forgot.

many people have considered the new flinch system incredibly disorienting and difficult to deal with in one on one PVP. a good amount of people have moved to servers that use custom plugins to entirely disable flinch because of motion sickness and/or similar issues.

**Comment by SDGNelson** [comment]
IIRC my hesitancy with adding an option for it was that it'd provide a competitive advantage. That said, accessibility and options to prevent motion sickness are very important. Do you feel motion sick from the damage flinch? Or a friend who feels motion sick?

**Reply by ghost** [reply]
i dont personally, but i play on a server which uses a plugin to disable it, and many of the regulars of that server are prone to motion sickness.
from what ive been told from said regulars, the current state of the flinch system actually killed off a big portion of the PVP side of the community. i think current flinching could be toned down a little overall to compensate?

i feel like theres also a tradeoff for disabling it per person, as while there isnt as much to obstruct aim, new flinching gives a good sense of direction for where youre being attacked from, perhaps a flinch multiplier setting could be in place for the client? letting people turn it down if they are prone to that kind of thing.
ill share this around to some of the people who are affected by it for a second opinion.

**Reply by ghost** [reply]
i think that also explosion wobble can be a bit much, and some of the things that can apply it in vanilla should be toned down. i think that only explosions that are typically linked to raiding (as a baseline) should give camera wobble, such as tank missiles, rocket launcher rockets, demo and precision charges - but not frag grenades, sticky grenades, makeshift grenades, fragmentation ammo and *maybe* bounce grenades and charges?

but im not 100% sure, some people would prefer it entirely disableable as its also a motion sickness trigger.
im in the boat of wanting both new flinch and explosion wobble on, because i, personally dont mind it, but i speak for others in this.

**Reply by M10327** [reply]
I get very bad motion sickness from the flinch and explosion wobble. I can only play on my own server where those are both disabled (via a plugin) because they make me dizzy and give me a headache within minutes. Any sort of camera movement that happens without my input (and *especially* random camera movement) is almost certain to give me a headache very quickly. Personally, I would like to see a server option to toggle these, or a client setting that allows you to toggle them and then servers can decide if they want to still force them on. A slider for intensity would also work, as long as we're allowed to drop it to zero. Though whatever you do, *please* do not make a change that prevents me from using a plugin to remove them both, because I would have to quit the game if I couldn't disable them.

**Reply by SDGNelson** [reply]
In this case: I'll plan to add some accessibility options for these in the next patch/update!
